src/HW/LedgerConnect/LedgerConnect.tsx

```diff
@@ -174,7 +174,7 @@ class _LedgerConnect extends React.Component<Props, State> {
         refreshing: false,
         waiting: true,
       })
-      await onConnectBLE(device.id.toString())
+      onConnectBLE(device.id.toString())
```

Comment:
`onConnectBLE` is an async function in `ConfirmTxWithHW.tsx`, will this not cause any issues? Eg the error handling below or animation (loading state). I remember we had a UI bug where we did not await an async function and this looks very similar. Maybe we should change the type of `onConnectBLE` to `onConnectBLE: (deviceId: DeviceId) => Promise<void> | void` and then `await onConnectBLE(device.id.toString()) `?

The same applies to `onConnectUSB`